.github/actions/setup-rocm/action.yml

```diff
@@ -71,12 +57,6 @@ runs:
       uses: ./.github/actions/diskspace-cleanup
       if: always()
 
-    - name: Runner health check disconnect on failure
-      if: ${{ failure() }}
-      shell: bash
-      run: |
-        killall runsvc.sh
-
```

Comment:
@saienduri Is this meant to be temporary? While some checks might need to be disabled for MI300 nodes, it seems like a good idea to still fail if the applicable health checks fail?

Reply:
Yes, meant to be temporary (deleted while dealing with other failures). Added back now, thanks